CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if (NOT DEFINED BUILD_NOSTATIC AND DEFINED ENV{BUILD_NOSTATIC})
   set(BUILD_NOSTATIC "$ENV{BUILD_NOSTATIC}") 
 endif()
 
+if (NOT DEFINED BUILD_NO32 AND DEFINED ENV{BUILD_NO32}) 
+  set(BUILD_NO32 "$ENV{BUILD_NOSTATIC}") 
```

Comment:
`set(BUILD_NO32 "$ENV{BUILD_NO32}")`?

Reply:
@agroce is this intentional?

Reply:
Oops no.  I will have to fix this whole mess up sometime soon